style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: poppins, serif;
+}
+
+main {
+  width: 375px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  min-width: 375px;
+  padding: 24px  24px;
+}
+
+section {
+  background-image: url(./images/header-shapes\ mobile.png);
+  background-repeat: no-repeat;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;

```

Comment:
- [ ] kindly remove the min-height so as to avoid the extra space at the bottom
- [ ] add background-size and set it to cover